cylc/flow/scripts/cylc_remote_init.py

```diff
@@ -55,8 +56,13 @@ def main():
         dest="indirect_comm",
         default=None,
     )
-    options, (install_target, rund) = parser.parse_args()
-    remote_init(install_target, rund, indirect_comm=options.indirect_comm)
+    options, (install_target, rund, *
+              dirs_to_be_symlinked) = parser.parse_args()
```

Comment:
I didn't know we could have arbitrary whitespace between the `*` and the variable, I do now!

Reply:
Me neither, maybe it would be better to keep them together for readability?

Reply:
This was what my auto formatter, I'll change it for readability.